### DISCUSSION TITLE
Is it possible to redefine any key? For example: the "eject" key.

### DISCUSSION THREAD

**Discussion by JoeZZDEV** [rxhanson/Multitouch-Community#153] [General]
I can't capture a specific key in the app to redefine it. Is it possible to add this feature of capturing key events?

**Comment by rxhanson** [comment]
At this time, Multitouch is unable to remap the "eject" key due limitations in macOS and how Multitouch performs the remapping. I would guess that Karabiner would be able to remap it, which works in a different way than Multitouch.

**Reply by JoeZZDEV** [reply]
Thanks.
Yes, Karabiner can do that.
I hope that I could replace the software with Multitouch which has similar functionality.
